content/en/docs/concepts/containers/images.md

```diff
@@ -10,7 +10,7 @@ hide_summary: true # Listed separately in section index
 
 <!-- overview -->
 
-A container image represents binary data that encapsulates an application and all its
+A container image is an OCI artifact that that encapsulates an application and all its
```

Comment:
This does need either a glossary tooltip or a plain hyperlink. Without it, we're making the page more confusing, not less.